hack/verify.sh

```diff
@@ -1,38 +1,44 @@
 #!/bin/bash
 
-export CRTDIR=$(pwd)
-export TMPDIR=/tmp/testgendocs
-mkdir $TMPDIR
+CRT_DIR=$(pwd)
+VERIFY_TEMP=$(mktemp -d 2>/dev/null || mktemp -d -t k8s-community.XXXXXX)
```

Comment:
I seem to remember that mktemp has different behavior between BSD/mac and GNU.   Worth testing.

Reply:
I grabbed this from the ensure-temp-dir function in k/k for just that reason:
https://github.com/kubernetes/kubernetes/blob/be8f1cea8082b03e4322f9f6b474f5ed5ff8679a/hack/lib/util.sh#L79-L84